platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -101,15 +101,15 @@ are arranged as a compelling internal product. Autonomous delivery teams can
 make use of the platform to deliver product features at a higher pace, with 
 reduced coordination."
 
-The specific set of capabilities and scenarios supported
-by a platform should be determined by the needs of your internal stakeholders. 
-And while platforms _provide_ these required capabilities, it's critical to note 
-that platform teams should not always _implement_ them themselves. Managed service 
-providers or dedicated internal teams can maintain backing implementations 
-while platforms are the thinnest reasonable layer that provides consistency for these 
-implementations that meets an organization's requirements. 
-For example, a very simple "platform" could be a wiki page with links to 
-standard operating procedures to provision capabilities from providers, as described in [[3]].
+The specific set of capabilities and scenarios supported by a platform should be
+determined by the needs of stakeholders and users. And while platforms _provide_
+these required capabilities, it's critical to note that platform teams should
+not always _implement_ them themselves. Managed service providers or dedicated
+internal teams can maintain backing implementations while platforms are the
+thinnest reasonable layer that provides consistency for these implementations
+that meets an organization's requirements.  For example, a very simple
```

Comment:
Small suggestion, in case the subject here is "these implementation"

```suggestion
that meet an organization's requirements.  For example, a very simple
```

Reply:
It is a bit hard to read as is. I changed to: "the thinnest reasonable layer that provides consistency across provided implementations and meets an organization's requirements"